tests/fiaas_deploy_daemon/conftest.py

```diff
@@ -187,6 +187,6 @@ def _mock_namespace_file_open(name, *args, **kwargs):
         yield mock_open
 
 
-@pytest.fixture(scope="session", params=("v1.9.11", "v1.10.12", "v1.12.10", "v1.14.4"))
+@pytest.fixture(scope="session", params=("v1.9.11", "v1.10.12", "v1.12.10", "v1.14.4", "v1.16.3"))
```

Comment:
Any chance we can drop 1.9.11 support? (FINN is on 1.12, moving to 1.14 and 1.16 soon)

Every added version increases the time to perform tests, so we don't want this list to grow too big.

Reply:
We still have a 1.9 cluster 😢 
Dropping 1.10 would be fine for us, we have 1.9 and 1.14 and the next ones will be 1.16 (or maybe higher, tbd...). What do you think?

Reply:
Let's drop 1.10 then. :+1:

Reply:
9a0430d does that